components/eamxx/src/share/grid/grids_manager.hpp

```diff
@@ -51,6 +51,12 @@ class GridsManager
 
   const grid_repo_type& get_repo () const { return m_grids; }
 
+  void print() const {
```

Comment:
This seems like a method you used while debugging, but no longer needed/used. Let's prune it.